runtime/doc/lua.txt

```diff
@@ -1195,12 +1203,11 @@ vim.bo[{bufnr}]                                                                *
         print(vim.bo.baz)                 -- error: invalid key
 <
 vim.wo[{winid}]                                                                *vim.wo*
-    Get or set window-scoped |local-options| for the window with handle {winid}.
-    If [{winid}] is omitted, use the current window. Invalid {winid} or key
-    is an error.
+    Get or set window-scoped |options| for the window with handle {winid}.
+    Like `:set`. If [{winid}] is omitted then the current window is used.
+    Invalid {winid} or key is an error.
 
-    This is a wrapper around |nvim_set_option_value()| and
-    |nvim_get_option_value()| with `opts = {scope = local, win = winid}` .
+    Note this does not access |local-options| (`:setlocal`) of a window.
```

Comment:
```suggestion
    Note: This does not access |local-options| (`:setlocal`) of a window.
```

Maybe add the `nvim_set_value_option` invocation for this so users know what to do if they need it?